aten/src/ATen/native/ConvertCooToCsr.cpp

```diff
@@ -0,0 +1,70 @@
+#include <ATen/ATen.h>
+#include <ATen/Dispatch.h>
+#include <ATen/Parallel.h>
+
+namespace at {
+namespace native {
+
+namespace {
+
+constexpr int64_t GRAIN_SIZE = at::internal::GRAIN_SIZE;
+
+template <typename input_t, typename output_t>
+void convert_coo_to_csr_cpu(Tensor& result, const Tensor& input, const int64_t size) {
```

Comment:
Prefer using const references for `Tensor` where possible. The data of `result` can be modified still.
```suggestion
void convert_coo_to_csr_cpu(const Tensor& result, const Tensor& input, const int64_t size) {
```